recipe/meta.yaml

```diff
@@ -52,7 +51,7 @@ test:
     - conda create -n testchild2 --yes
     - conda info
     - conda info --envs
-    - python -m anaconda_anon_usage.install --status  # [variant=="patch"]
+    - python -m anaconda_anon_usage.install --expect  # [variant=="patch"]
```

Comment:
This affirmatively verifies that the package is properly installed and activated